wordpress-mobile/WordPress-iOS
# Integrate AztecEditor-iOS 1.19.4

Integrate AztecEditor-iOS 1.19.4 release.

* **Gutenberg:** WordPress/gutenberg#29355
* **Gutenberg Mobile:** wordpress-mobile/gutenberg-mobile#3187

To test:
- Verify CI checks pass.
- Verify Gutenberg and Classic editor open without issue.

PR submission checklist:
